examples/sagemaker-pipelines-graphbolt/README.md

```diff
@@ -0,0 +1,540 @@
+# Faster distributed graph neural network training with GraphStorm 0.4
+
+GraphStorm is a low-code enterprise graph machine learning (ML) framework that provides ML practitioners a simple way of building, training and deploying graph ML solutions on industry-scale graph data. While GraphStorm can run efficiently on single instances for small graphs, it truly shines when scaling to enterprise-level graphs in distributed mode using a cluster of EC2 instances or Amazon SageMaker.
+
+GraphStorm 0.4 introduced integration with DGL-GraphBolt, a new graph storage and sampling framework that uses a compact graph representation and pipelined sampling to reduce memory requirements and speed up Graph Neural Network (GNN) training. In this example we'll show how GraphStorm 0.4 brings training and inference speedups of up to 3x on the papers100M dataset.
```

Comment:
brings training and inference speedups of up to 1.4X and 3.6X respectively on the papers100M dataset.